common/src/test/java/org/jboss/aerogear/unifiedpush/system/ConfigurationUtilsTest.java

```diff
@@ -37,28 +37,28 @@ public void cleanupTestProperty() {
     @Test
     public void testExistingTryGetProperty(){
         System.setProperty(TEST_PROPERTY_NAME, "MyNiceValue");
-        assertThat(ConfigurationUtils.tryGetProperty(TEST_PROPERTY_NAME)).isEqualTo("MyNiceValue");
+        assertThat(ConfigurationUtils.tryGetGlobalProperty(TEST_PROPERTY_NAME)).isEqualTo("MyNiceValue");
```

Comment:
This all only tests the existing System properties, from the test.

Let's see if we can actually test ENV vars, includinge the fancy "toUpperCase and replaceAll" code.

I think it's possible to configure ENV_VARs in the pom file (sure fire plugin), like here:
https://github.com/aerogear/kafka-cdi/blob/master/pom.xml#L65-L77

but instead `systemPropertyVariables` it needs to be `environmentVariables`

Reply:
@matzew Yes definitely makes sense to add a test here, will do. Unfortunately envirnonment variables can't easily be set from Java so i will need some mocking.

Reply:
@pb82 I think, in the POM, on the surefire plugin they can be set - just a a few - I can take a look too, if you want